specification/portalservices/resource-manager/readme.md

```diff
@@ -27,7 +27,34 @@ These are the global settings for the portal.
 ```yaml
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2023-01-01-preview
+tag: package-2024-04-01-preview
+```
+
+### Tag: package-2024-04-01-preview
+
+These settings apply only when `--tag=package-2024-04-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-04-01-preview'
+input-file:
+  - Microsoft.PortalServices/copilotSettings/preview/2024-04-01-preview/copilotSettings.json
+  - Microsoft.PortalServices/extensions/preview/2023-01-01-preview/extensions.json
+suppressions:
```

Comment:
Could you mention the paths in "where" for each of the suppressions, instead of suppressing for the whole file.